renderer/tdf/android/src/main/java/com/tencent/renderer/TDFRenderer.java

```diff
@@ -114,12 +120,38 @@ public void init(@Nullable List<Class<?>> controllers, @Nullable ViewGroup rootV
         }
     }
 
+    @Override public void destroyRoot(int rootId) {
```

Comment:
`@Override` annotation move to previous line

Reply:
fixed